js/core.js

```diff
@@ -4332,7 +4332,7 @@
 
         getDisplay: function(context) {
             if (!(context in this._display)) {
-                var self = this, annotation = -1, columnOrder = [], showFKLink = true;
+                var self = this, annotation = -1, columnOrder = [], showFKLink = true, inputDisplayMode = 'popup', toTableAnnotation = -1;
```

Comment:
Let's use the constant that you defined here (`module._foreignKeyInputModes[0]`) so you don't have to hardcode any values.